src/shared_modules/content_manager/src/components/componentsHelper.hpp

```diff
@@ -0,0 +1,62 @@
+/*
+ * Wazuh Content Manager
+ * Copyright (C) 2015, Wazuh Inc.
+ * Nov 29, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef _COMPONENTS_HELPER_HPP
+#define _COMPONENTS_HELPER_HPP
+
+#include "json.hpp"
+#include "updaterContext.hpp"
+#include <map>
+#include <stdexcept>
+#include <string>
+
+namespace Utils
```

Comment:
I feel this helper function is not generic enough for this namespace. Searching for "namespace Utils" in the codebase shows some patterns base implementation, file manipulation classes, etc.
But, not sure what would be a better name. What is more related to? updater? pub-sub?
What do you think?

Reply:
Yes, it's true. I made some renaming.